core-web/libs/block-editor/src/lib/nodes/image-node/image.node.ts

```diff
@@ -99,7 +99,8 @@ export const ImageNode = Image.extend({
                     const { head } = selection;
                     const node = {
                         attrs: getImageAttr(attrs),
-                        type: ImageNode.name
+                        type: ImageNode.name,
+                        storage: { aiStatus: 'pending' }
```

Comment:
Is there an `Enum` for this?

Reply:
is not, and actually that need to be removed.